runescape-client/src/main/java/AbstractByteArrayCopier.java

```diff
@@ -88,7 +88,8 @@ static void changeWorld(World var0) {
 		descriptor = "(II)V",
 		garbageValue = "407362763"
 	)
-	static void method4087(int var0) {
+	@Export("setTempMenuAction")
+	static void setTempMenuAction(int var0) {
```

Comment:
Is it a problem to export & hook this method? It seems like a candidate for inlining perhaps.

Reply:
Discussed on discord and this will have to be done differently